nimble/transport/socket/src/ble_hci_socket.c

```diff
@@ -142,27 +144,27 @@ STATS_NAME_END(hci_sock_stats)
 #define BLE_SOCK_STACK_SIZE         \
     OS_STACK_ALIGN(MYNEWT_VAL(BLE_SOCK_STACK_SIZE))
 
-struct os_task ble_sock_task;
+struct ble_npl_task ble_sock_task;
 
-static struct os_mempool ble_hci_sock_evt_hi_pool;
+static struct os_mempool s_hci_sock_evt_hi_pool;
```

Comment:
why need to change static variable names?

Reply:
I thought `s_` was the naming convention used for static variables within nimble.  It looks like the convention may be `g_`.  Either way, when working in this code, I was running into cases where simple searches would not disambiguate between static variables and function names, so I moved these statics to follow a more common naming convention.